compiler/circle-opselector/src/OpSelector.cpp

```diff
@@ -0,0 +1,396 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "OpSelector.h"
+
+#include <luci/Import/GraphBuilder.h>
+#include <luci/Import/GraphBuilderContext.h>
+#include <luci/Import/GraphBuilderRegistry.h>
+#include <luci/Import/CircleReader.h>
+#include <luci/Import/Nodes/CircleConst.h>
+
+#include <luci/Profile/CircleNodeID.h>
+#include <luci/Profile/CircleNodeOrigin.h>
+
+#include <loco/IR/Graph.h>
+
+#include <oops/UserExn.h>
+
+#include <cassert>
+#include <iostream>
+#include <map>
+
+#define MAIN_SUBGRAPH 0
+
+// TODO: Re-implement to use just module.
+namespace opselector
+{
+
+void OpSelector::check_connected(std::vector<const luci::CircleNode *> &selected_nodes,
```

Comment:
Please add some doxygen format comments about this method for better understanding

Reply:
@dongyooncho I don't think adding doxygen comments will do. Better way is to rename the method name. I also expect `check_...` will return `true`/`false` or some enumeration and so on. But it returns nothing.